yarn.lock

```diff
@@ -25161,14 +25188,14 @@ __metadata:
   languageName: node
   linkType: hard
 
-"react-router@npm:6.17.0":
```

Comment:
Any idea why this was downgraded @ashharrison90?

Reply:
@zoltanbedi oh... no tbh 😕 all i did was run `yarn upgrade-interactive` and picked `react-redux`, not sure why it would have downgraded this 🤔